R/rmarkdown.R

```diff
@@ -0,0 +1,41 @@
+
+#' Add an RMarkdown Template
+#'
+#' @param template_dir Name of the directory the template will live in within
+#' \code{inst/rmarkdown/templates}.
+#' @param template_name The name as printed in the template menu.
+#' @param template_description A short description of the template.
+#' @param template_create_dir Sets the value of \code{create_dir}
+#' in the template.yml
+#' @inheritParams use_directory
+#'
+#' @export
+#' @examples
+#' \dontrun{
+#' use_rmarkdown_template()
+#' }
+use_rmarkdown_template <- function(template_dir = "newtemplate",
+                                   template_name = "Template Name",
+                                   template_description = "A description of the template",
+                                   template_create_dir = FALSE,
+                                   base_path = ".") {
+
+  # Process some of the inputs
+  template_create_dir <- if(isTRUE(template_create_dir)) "true" else "false"
+  template_dir <- file.path("inst", "rmarkdown", "templates", template_dir)
+
+  # Scaffold files
+  use_directory(file.path(template_dir, "skeleton"), base_path = base_path)
+  use_template("rmarkdown-template.yml", data = list(template_dir = template_dir,
```

Comment:
Can you please use tidyverse style for long lines? http://style.tidyverse.org/syntax.html#long-lines